numpy/lib/tests/test_polynomial.py

```diff
@@ -243,6 +243,12 @@ def test_polydiv(self):
         assert_equal(q.coeffs.dtype, np.complex128)
         assert_equal(r.coeffs.dtype, np.complex128)
         assert_equal(q*a + r, b)
+        
+        d = np.poly1d([1, 2, 3])
+        c = [1, 2, 3]
+        s, t = np.polydiv(d, c)
```

Comment:
This test is backwards - its the one that passed before this patch.

Probably fine to keep it, but you should add one with the arguments in the other order too.